src/handlers.ts

```diff
@@ -82,11 +82,23 @@ export async function handleChromiumCheck(): Promise<void> {
     }
 
     d(`Computing latest upstream version for Chromium ${chromiumMajorVersion}`);
-    const upstreamVersions = Object.keys(chromiumTags)
-      .filter(v => Number(v.split('.')[0]) === chromiumMajorVersion)
-      // NB. Chromium rolled a 3905 branch on m78 but abandoned it and continued with 3904.
-      .filter(v => !v.startsWith('78.0.3905.'))
-      .sort(compareChromiumVersions);
+    const relevantChromiumReleases = chromiumReleases.filter(
+      r => r.os === 'win' || r.os === 'win64' || r.os === 'mac' || r.os === 'linux',
```

Comment:
nit: shorter to do: `['win', 'win64', 'mac','linux'].includes(r.os)`

Reply:
made it even shorter 🏌️